browser/app/js/components/App.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Minio Cloud Storage (C) 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import { Route, Switch, Redirect } from "react-router-dom"
+import Browser from "../components/Browser"
+import Login from "../components/Login"
+import web from "../web"
+import { minioBrowserPrefix } from "../constants"
+
+const AutherizedRoute = ({ component: Component, ...rest }) => (
```

Comment:
Nothing too important, but we have a very small typo here, it should be `AuthorizedRoute` instead of `AutherizedRoute` (with an "o" instead of the first "e"). The same applies for lines 39, 41, and 42 as well.

Reply:
Will fix it

Reply:
@kaankabalak its fixed, please review